### PR TITLE
readthedocs yaml configuration

### PR DESCRIPTION
**Pull request by ShadenSmith**

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/code-docs/source/conf.py
+  fail_on_warning: false
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+  - pdf
```
Comment (jeffra) [inline .readthedocs.yml:12]: Will this actually build the docs somewhere in PDF format? interesting

**Review by jeffra — APPROVED** [review]

**Review by arashashari — APPROVED** [review]
Please check the failing test, otherwise LGTM, and thanks for handling it.

**Comment by jeffra** [comment]
Seems it's failing with this import for some reason: https://github.com/microsoft/DeepSpeed/blob/91b4a93db01318cf7ad39b33ee3b2d2715abac54/deepspeed/ops/__init__.py#L1

File: .readthedocs.yml (hunk 1 above)
Comment (ShadenSmith) [reply .readthedocs.yml:12]: Yep! https://docs.readthedocs.io/en/stable/downloadable-documentation.html

**Pull request merged**